libs/core/langchain_core/language_models/chat_models.py

```diff
@@ -304,6 +316,39 @@ async def ainvoke(
         )
         return cast(ChatGeneration, llm_result.generations[0][0]).message
 
+    def _should_stream(
+        self,
+        *,
+        async_api: bool,
+        run_manager: Optional[
+            Union[CallbackManagerForLLMRun, AsyncCallbackManagerForLLMRun]
+        ] = None,
+        **kwargs: Any,
+    ) -> bool:
```

Comment:
Would a bit of documentation here be helpful to explain WHY one might need to disable streaming?